05_matplotlib.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Formative Assessment 2:</p><p> </p><p>What happens if you do not include plt.legend() in the code above? ¿Qué sucede si no incluye plt.legend() en el código anterior?</p><p> </p><p>1) Nothing / Nada</p><p>2) The legend does not show in the figure / La leyenda no aparece en la figura.</p><p>3) The legend still shows in the figure / La leyenda aún se muestra en la figura.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CodeToCommunicate/CoCoLessons/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>